.github/workflows/integrate.yaml

```
@@ -38,6 +38,17 @@ jobs:
       - run: sudo apt update && sudo apt install tox
       - run: tox -e ${{ matrix.charm }}-unit
 
+  terraform-checks:
+    name: Terraform
+    uses: canonical/charmed-kubeflow-workflows/.github/workflows/terraform-checks.yaml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        charm: [operator, serving, eventing]
+    with:
+      charm-path: ./charms/knative-${{ matrix.charm }}
+      apply: false
```

Maybe add a simple comment on why we don't this workflow here
```suggestion
      # We don't run this workflow since the charm goes to `Error` state when deployed on its own
      # See: <insert_link_to_issue>
      apply: false
```

---

thank you, updated